services/notifications/README.md

```diff
@@ -35,6 +43,7 @@ templates
 ```
 
 Custom email templates referenced via `NOTIFICATIONS_EMAIL_TEMPLATE_PATH` must also be located in subfolders `shares` and `spaces` and must have the same names as the embedded templates. It is important that the names of these files and  folders match the embedded ones.
+In the subfolder `html` contains a default HTML template provided by ocis. The images can be embedded in a custom HTML template as a CID source ```<img src="cid:logo-mail.gif" alt="logo-mail"/>``` The image files should be located in `html/img` subfolder. Supported image types are png, jpeg, and gif.
```

Comment:
If I have `NOTIFICATIONS_EMAIL_TEMPLATE_PATH` set. Will it look in `$NOTIFICATIONS_EMAIL_TEMPLATE_PATH/html/img` ?

Reply:
Yes, it pointed on  `$NOTIFICATIONS_EMAIL_TEMPLATE_PATH/html/img`